src/CodeGeneration.Roslyn.Tests.Generators.Dependency/CodeGeneration.Roslyn.Tests.Generators.Dependency.csproj

```diff
@@ -2,7 +2,6 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard1.6</TargetFramework>
```

Comment:
Is it intended to leave 1.6 here?

Reply:
Yes, since it's a dependency *of* the generator, it can be left, it shouldn't matter.